mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -162,7 +163,6 @@ CLASS net/minecraft/unmapped/C_ayfeobid net/minecraft/client/MinecraftClient
 	FIELD f_yiluvghr targetedEntity Lnet/minecraft/unmapped/C_astfners;
 	FIELD f_yjhobbqr renderTickCounter Lnet/minecraft/unmapped/C_ewnwsuig;
 	FIELD f_ypxesqvh instance Lnet/minecraft/unmapped/C_ayfeobid;
-	FIELD f_ypxxahyx LOGGER Lorg/slf4j/Logger;
```

Comment:
Why loggern't?

Reply:
Why PR description readn't? :(

(the variable is auto-mapped, so we don't have to keep the specific mapping, Enigma has it automatically)

Reply:
I can seen't. my bad